monster-cards/src/widgets/card/createCardSummary.ts

```diff
@@ -1,50 +1,34 @@
-import createRenderMixin, { RenderMixin, RenderMixinOptions, RenderMixinState } from 'dojo-widgets/mixins/createRenderMixin';
-import createRenderableChildrenMixin from 'dojo-widgets/mixins/createRenderableChildrenMixin';
-import createParentListMixin, { ParentListMixin, ParentListMixinOptions } from 'dojo-widgets/mixins/createParentListMixin';
+import { Widget, DNode } from 'dojo-interfaces/widgetBases';
+import createWidgetBase from 'dojo-widgets/bases/createWidgetBase';
+import d from 'dojo-widgets/util/d';
 import createCard from './createCard';
-import createWidget from 'dojo-widgets/createWidget';
-import { Child } from 'dojo-widgets/mixins/interfaces';
 
-export type CardSummaryState = RenderMixinState & {
+export type CardSummaryState = {
 	name: string;
 	imageClass: string;
 	score: number;
 	cardId: string;
 }
 
-type CardSummaryOptions = RenderMixinOptions<CardSummaryState> & ParentListMixinOptions<Child>;
+export type CardSummary = Widget<CardSummaryState>
 
-export type CardSummary = RenderMixin<CardSummaryState> & ParentListMixin<Child>;
+const createCardSummary = createWidgetBase.extend({
+	tagName: 'div.cardSummary',
```

Comment:
While this works, it's probably better to use the `classes` array in the extends.